website/content/configuration/_index.md

```diff
@@ -23,6 +23,10 @@ to announce service IPs. Jump to:
 Layer 2 mode is the simplest to configure: in many cases, you don't
 need any protocol-specific configuration, only IP addresses.
 
+Layer 2 mode does not require the IPs to be bound to the network interfaces
+of your worker nodes. It works by _flooding_ your local network with ARP
```

Comment:
Flooding is not the right term here (sounds negative, like a DoS attack). How about:

It works by responding to ARP requests on your local network directly, to give the machine's MAC address to clients.